content/en/blog/2022-08-05-effective-programmers-read-code.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Effective Programmers Read Code"
+slug: "effective-programmers-read-code"
+date: "2022-08-05T08:10:14-04:00"
+draft: true
+image_webp: images/blog/reading.webp
+image: images/blog/reading.jpg
+author: Benjamin Bengfort
+description: "While we often think ourselves as authors of code we probably spend more time reading code than writing it. Changing our mindset to think of ourselves as avid code readers will make us more effective programmers."
+---
+
+Programmers think of themselves as authors of code but our daily engineering practice probably requires us to spend more time reading code than writing it. Slightly changing our mindset to think of ourselves as avid code readers will make us more effective programmers and ensure that we can quickly create mental maps of our code, a technique that I refer to as the "glass castle".
```

Comment:
Suggestion: As programmers, we tend to think of ourselves as authors of code, but our daily engineering practice...